tree/ntuple/v7/inc/ROOT/RNTuple.hxx

```diff
@@ -180,10 +180,9 @@ public:
    static std::unique_ptr<RNTupleReader> Open(std::string_view ntupleName,
                                               std::string_view storage,
                                               const RNTupleReadOptions &options = RNTupleReadOptions());
+   /// The caller imposes a model, which must be compatible with the model found in the data on storage.
    static std::unique_ptr<RNTupleReader>
-   Open(RNTuple *ntuple, const RNTupleReadOptions &options =
-                            RNTupleReadOptions()); /// The caller imposes a model, which must be compatible with the
-                                                   /// model found in the data on storage.
+   Open(RNTuple *ntuple, const RNTupleReadOptions &options = RNTupleReadOptions());
    static std::unique_ptr<RNTupleReader> Open(std::unique_ptr<RNTupleModel> model, std::string_view ntupleName,
                                               std::string_view storage,
                                               const RNTupleReadOptions &options = RNTupleReadOptions());
```

Comment:
I guess the comment should go to the `Open` function that actually takes a model...